src/test/java/teammates/test/cases/logic/TeamEvalResultTest.java

```diff
@@ -491,7 +491,7 @@ private void verifyNormalized(double[] expected, double[] input) {
                 Arrays.toString(TeamEvalResult.normalizeValues(input)));
     }
     
-    private static void showCalculationSteps(int[][] input) {
+    /*private static void showCalculationSteps(int[][] input) {
```

Comment:
Hmm, we could try that. This test file doesn't seem like the right place for it anyway.